src/dict_sorting.jl

```diff
@@ -18,7 +18,6 @@ function sort!(d::OrderedDict; byvalue::Bool=false, args...)
 end
 
 sort(d::OrderedDict; args...) = sort!(copy(d); args...)
-sort(d::Dict; args...) = sort!(OrderedDict(d); args...)
```

Comment:
Come to think of it, given how widely used this package is, perhaps we should change this to
```julia
@deprecate sort(d::Dict; args...) sort!(OrderedDict(d); args...)
```
(GitHub doesn't let me make mergeable suggestions for deleted lines.)

Reply:
I had to do this using the Github editor. Hope it's ok.